### PR TITLE
Add reliquarry support

### LINKED ISSUE #95: Reliquarry support missing

### PR DESCRIPTION
**Pull request by EndangeredNayla**
Fixes #95

### CODE REVIEW

File: src/main/java/com/kuba6000/mobsinfo/loader/extras/Reliquarry.java [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            addDrop(drops, XRRecipes.ingredient(Reference.BAT_INGREDIENT_META), Names.bat_wing);
+        } else if (recipe.entity instanceof EntitySnowman) {
+            addDrop(drops, XRRecipes.ingredient(Reference.FROZEN_INGREDIENT_META), Names.frozen_core);
+        }
+    }
+
+    private void addDrop(ArrayList<MobDrop> drops, ItemStack item, String name) {
+        MobDrop drop = new MobDrop(item, MobDrop.DropType.Normal, 0, null, null, false, false);
+        drop.variableChance = true;
+        drop.chanceModifiers.add(new ReliquaryDropChance(name, eventHandler));
```
Comment (kuba6000) [inline src/main/java/com/kuba6000/mobsinfo/loader/extras/Reliquarry.java:91]: Is there anything special about that drop chance? The only thing I see in the description is the chance. If thats the case, there is no need for that
```suggestion
        MobDrop drop = new MobDrop(item, MobDrop.DropType.Normal, (int)(eventHandler.getBaseDrop(name) * 10000d), null, null, false, false);
```

File: src/main/java/com/kuba6000/mobsinfo/loader/extras/Reliquarry.java [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        drops.add(drop);
+    }
+
+    private static class ReliquaryDropChance implements IChanceModifier, Serializable {
```
Comment (kuba6000) [inline src/main/java/com/kuba6000/mobsinfo/loader/extras/Reliquarry.java:95]: Special class is not needed if there is nothing special about the drop

**Review by kuba6000 — CHANGES_REQUESTED** [review]

File: src/main/java/com/kuba6000/mobsinfo/loader/extras/Reliquarry.java (hunk 2 above)
Comment (EndangeredNayla) [reply src/main/java/com/kuba6000/mobsinfo/loader/extras/Reliquarry.java:95]: special class?

**Comment by kuba6000** [comment]
@EndangeredNayla Can you apply spotless? `gradlew spotlessApply`

**Review by kuba6000 — APPROVED** [review]
Thanks

**Pull request merged**